solidity/contracts/BancorNetwork.sol

```diff
@@ -81,7 +81,7 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, R
       * @param _registry    address of a contract registry contract
     */
     constructor(IContractRegistry _registry) ContractRegistryClient(_registry) public {
-        etherTokens[ETH_RESERVE_ADDRESS] = true;
+        etherTokens[IERC20Token(ETH_RESERVE_ADDRESS)] = true;

```

Comment:
If we happen to use `ETH_RESERVE_ADDRESS` as `IERC20Token` in every place in this file, then you may want to consider declaring it as `IERC20Token` to begin with.

Reply:
That's a good suggestion

Reply:
thought it might be a bit too much, you guys sure about it?

Reply:
done